### ISSUE TITLE
test has dependency on fbpic

### ISSUE THREAD

**Issue by s9105947** [picmi-standard/picmi#62]
When executing the test it reports a dependency on fbpic.
Should this test be moved to fbpic?

```
$ ./launch_test.sh
Traceback (most recent call last):
  File "../Examples/laser_acceleration/laser_acceleration_PICMI.py", line 5, in <module>
    from fbpic import picmi
ModuleNotFoundError: No module named 'fbpic'
```

**Comment by dpgrote** [comment]
Thanks for pointing this out. I'll submit a PR to fix this so that it uses the dummy `plasmacode` instead.

**Comment by ax3l** [comment]
Fixed via  #64
